src/main/resources/alfresco-global.properties

```diff
@@ -171,3 +171,7 @@ content.transformer.arender2pdf.extensions.xltx.pdf.priority=30
 
 content.transformer.arender2pdf.extensions.zip.pdf.supported=true
 content.transformer.arender2pdf.extensions.zip.pdf.priority=30
+
```

Comment:
You have 2 copies of this file laying around: one here and one in `src/main/amp/config`.
We should only keep the one in `src/main/amp/config`, and this one should be deleted.

This file is located in the root of the jar file inside the amp, and will be ignored anyways.